tools/fast/tests/slashing_test.go

```diff
@@ -0,0 +1,125 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"testing"
+	"time"
+
+	files "github.com/ipfs/go-ipfs-files"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-filecoin/actor/builtin/miner"
+	"github.com/filecoin-project/go-filecoin/address"
+	"github.com/filecoin-project/go-filecoin/protocol/storage/storagedeal"
+	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
+	"github.com/filecoin-project/go-filecoin/tools/fast"
+	"github.com/filecoin-project/go-filecoin/tools/fast/fastesting"
+	"github.com/filecoin-project/go-filecoin/tools/fast/series"
+	"github.com/filecoin-project/go-filecoin/types"
+)
+
+// Not passing, and also too slow to let run in CI
+func TestSlashing(t *testing.T) {
+	tf.IntegrationTest(t)
+	t.SkipNow()
```

Comment:
This proves that slashing works, but it runs way too long for CI.

Reply:
If it takes too long, it should be a functional test.

Reply:
True, and I fear that it will still make CI unacceptably long, but we'll try it.